src/installation/install_epinio_on_rke.md

```diff
@@ -6,6 +6,7 @@ If you don't have an existing cluster, follow the [quickstart](https://docs.rke2
 
 ## RKE2 Prerequisites
 
+* [MetalLB](https://metallb.universe.tf) was used as load-balancer
```

Comment:
If RKE2 is installed on public clouds, like AWS, GCP etc... MetalLB is not required as the cloud providers often have its own load balancer. 
MetalLB should be used when we can't use embedded load balancer. WDYT?

Reply:
I would replace the comma in `work on RKE2, you can` with a period (`.`). IOW start a new sentence.